src/GoogleLogin.js

```diff
@@ -1,54 +1,64 @@
 import React from 'react';
 
-export default class GoogleLogin extends React.Component{
-  constructor(props) {
+var gapi;
+var auth2;
+
+export default class GoogleLogin extends React.Component {
+  constructor (props) {
     super(props);
   }
 
   componentDidMount () {
-    (function(d, s, id){
-     var js, gs = d.getElementsByTagName(s)[0];
-     if (d.getElementById(id)) {return;}
-     js = d.createElement(s); js.id = id;
-     js.src = 'https://apis.google.com/js/platform.js'
-     gs.parentNode.insertBefore(js, gs);
-   }(document, 'script', 'google-platform'));
+    window.googlePlatformReadyCb = () => {
+      console.info('gapi ready.');
```

Comment:
sorry @kennetpostigo was a bit busy and didn't keep track of the thread and thanks @D-Andreev for checking my changes